docs/theming/dark-mode.md

```diff
@@ -117,96 +210,246 @@ For developers looking to customize the theme color under the status bar in Safa
 
 ## Ionic Dark Theme
 
-Ionic has a recommended theme for variables to use in order to get a dark mode based on the device running the app. It can be broken down into the following parts:
+Ionic has a recommended dark theme that can be enabled in three different ways: [always](#always), based on [system](#system) settings, or by using a [class](#class). Each of these methods involves importing the dark theme file with the corresponding name.
+
+The contents of each file are included below for reference. These variables are set by importing the relevant dark theme file and do not need to be copied into an app. For more information on the variables being changed, including additional variables for further customization, refer to the [Themes](themes.md) section.
+
+<Tabs groupId="darkFiles" defaultValue="always" values={[{ value: 'always', label: 'Always (dark.always.css)' }, { value: 'system', label: 'System (dark.system.css)' }, { value: 'class', label: 'Class (dark.class.css)' }]}>
+
+<TabItem value="always">
+
+The **always** dark theme behaves in the following ways:
+
+1. Sets the [Ionic colors](colors.md) for all [modes](platform-styles.md#ionic-modes) to complement a dark theme in the `:root` selector. The [`:root`](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) selector is identical to the selector `html`, except that its [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) is higher.
+2. Setting variables for the dark theme on `ios` devices using the `:root.ios` selector.
+3. Setting variables for the dark theme on `md` devices using the `:root.md` selector.
+
+:::caution
+It is important to pay attention to the specificity if you want to override any of the Ionic dark theme variables. For example, because the `--ion-item-background` variable is set for each mode, it cannot be overridden in the `:root` selector. A higher specificity selector, such as `:root.ios`, is required.
+:::
+
+```css
+:root {
+  --ion-color-primary: #4d8dff;
```

Comment:
I copied and pasted the dark theme variables from the framework here: https://github.com/ionic-team/ionic-framework/blob/feature-8.0/core/src/css/themes/dark.scss

I believe it would be better to import the file contents [from the CDN](https://cdn.jsdelivr.net/npm/@ionic/core@7.6.2-dev.11705355381.14b22962/css/themes/dark.class.css) to keep this section up to date with the latest version. However, I chose not to do so as it would expand the scope of this work.

Reply:
Could we provide a smaller example so we don't need to be maintaining this code snippet? If we keep it open ended then we don't have to worry about having to update the hex colors.

```css
:root {
  --ion-color-primary: // color provided by Ionic
}

:root.ios {
  --ion-color-primary: // your preferred color
}

:root.md {
  --ion-color-primary: // your preferred color
}
```

Reply:
It isn't supposed to be an example of setting a dark theme; instead, it is meant to serve as a reference for the entire Ionic dark theme we provide. I tried to override an application variable (`--ion-toolbar-background`) in `:root`, but it didn't work because the `md` theme was overriding it. As a result, I had to look up the defaults to see which variables are overridden by the theme. Providing developers with the entire theme file gives them the ability to see where we are setting the variables and how to override them.

Reply:
Why not point to the repo file?

Reply:
I'm not sure how to do that in a way that it would stand out and not be missed. I feel like links to the files would be easy to overlook. Ultimately I think we should have some kind of a preview widget where we could just include the CDN URL and then it would display the contents, but we would need to add that feature.

Reply:
Ah, that makes sense. I can't think of a different alternative so this works.